collectors/okta/okta_collector.js

```diff
@@ -75,9 +76,12 @@ class OktaCollector extends PawsCollector {
         })
         .catch((error) => {
             error.errorCode = this._isNoErrorCode(error);
-            if (this._isThrottlingError(error)) {
-                collector.reportApiThrottling(function() {
-                    return callback(error);
+            let isThrottle = this._isThrottlingError(error, state);
```

Comment:
just a cosmetic really, but can we rename `_isThrottlingError` and `isThrottle`? the `isX` convention is usually reserved for boolean types

maybe something like
```javascript
let maybeThrottlingError = this._resolveErrorType(error, state);
if (maybeThrottlingError.rateLimitError) {
  // ...
}
```